_targets.R

```diff
@@ -6,37 +6,53 @@ source("3_visualize/src/plot_timeseries.R")
 options(tidyverse.quiet = TRUE)
 tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse because we need dplyr, ggplot2, readr, stringr, and purrr
 
+site_nums <- c("01427207","01432160","01435000","01436690","01466500")
+
+p_width <- 12
+p_height <- 7
+p_units <- "in"
+
 p1_targets_list <- list(
   tar_target(
-    site_data,
-    download_nwis_data(),
-  ),
+    nwis_01427207_csv,
+    download_nwis_site_data(file_save_path = "1_fetch/out",site_num = site_nums[1]),
+    format="file"),
+  tar_target(
+    nwis_01432160_csv,
+    download_nwis_site_data(file_save_path = "1_fetch/out",site_num = site_nums[2]),
+    format="file"),
+  tar_target(
+    nwis_01435000_csv,
+    download_nwis_site_data(file_save_path = "1_fetch/out",site_num = site_nums[3]),
+    format="file"),
+  tar_target(
+    nwis_01436690_csv,
+    download_nwis_site_data(file_save_path = "1_fetch/out",site_num = site_nums[4]),
+    format="file"),
+  tar_target(
+    nwis_01466500_csv,
+    download_nwis_site_data(file_save_path = "1_fetch/out",site_num = site_nums[5]),
+    format="file"),
+  tar_target(
+    combined_site_data, 
+    combine_site_data(c(nwis_01427207_csv,nwis_01432160_csv,nwis_01435000_csv,nwis_01436690_csv,nwis_01466500_csv))),
   tar_target(
     site_info_csv,
-    nwis_site_info(fileout = "1_fetch/out/site_info.csv", site_data),
-    format = "file"
-  )
+    nwis_site_info(fileout = "1_fetch/out/site_info.csv",combined_site_data))
```

Comment:
This target here is generating an output file so you want to use the 'format="file"' option here